developer_cloudslang.rst

```diff
@@ -25,7 +25,7 @@ Embed CloudSlang in a Java Application
     <dependency>
       <groupId>io.cloudslang.lang</groupId>
       <artifactId>cloudslang-all</artifactId>
-      <version>0.8</version>
+      <version>0.9.50.0</version>
```

Comment:
but here we need to be careful -- if we update, we need to retest
I think it's ok to stay with this version for now and update `cloudslang-all` and `score-all` dependencies when releasing   `1.0`

.. I am reverting this line ..